resources/lang/en.json

```diff
@@ -127,6 +128,7 @@
   "Client ID": "Client ID",
   "Client Secret": "Client Secret",
   "Close": "Close",
+  "CLOSE": "CLOSE",
```

Comment:
I don't think we should have different translation for an all-caps version of the word "close". I think the translation above this one ("Close") should be used, and a style of `text-transform: capitalize;` should be applied to it. There is a bootstrap helper for this too: https://getbootstrap.com/docs/4.0/utilities/text/#text-transform.

```
<p class="text-lowercase">Lowercased text.</p>
<p class="text-uppercase">Uppercased text.</p>
<p class="text-capitalize">CapiTaliZed text.</p>
```

Reply:
I thought I found `CLOSE` in screen builder, but when I change it and link it into processmaker it doesn't change.